binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,182 @@
+# Node for Binary Search Tree
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+# Function for find the left side right most node in binary search tree
+def find_leftside_rightmost(node):
+    left_side_rightmost = node.left
+    prev_node = node
+    while left_side_rightmost.right is not None:
+        prev_node = left_side_rightmost
+        left_side_rightmost = left_side_rightmost.right
+    return left_side_rightmost, prev_node
+
+# Binary Search Tree declaration
+class BST:
+    def __init__(self):
+
+        self.root = None  # Root for Binary Search Tree
+        self.count = None  # Size of Binary Search Tree
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpz8i066jt/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmpz8i066jt/binary_search_tree/BinarySearchTree.py
@@ -25,7 +25,7 @@
     def insert(self, data):
 
         new_node = Node(data)  # Create a node for the new entry
-        
+
         # Insert a node to empty BST
         if self.root is None:
             self.root = new_node
```